includes/wikia/api/DWDimensionApiControllerSQL.class.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+class DWDimensionApiControllerSQL {
```

Comment:
@macbre Is this a reasonable way to include SQL or is there some other preferred way?